### PR TITLE
Use sbt-org-policies plugin

### LINKED ISSUE #4: Update dependency org.specs2:specs2-core:test
Actual version: 4.2.0
 Patch: -
 Minor: 4.3.2
 Major: -

### PR DESCRIPTION
**Pull request by franciscodr**
Once `skeuomorph` has been moved from [frees-io](https://github.com/frees-io) to [higherkindness](https://github.com/higherkindness) organization, does it make sense to start using `sbt-org-policies` instead of `sbt-freestyle` plugin?

It closes #4, #5 and #6

### CODE REVIEW

**Comment by franciscodr** [comment]
Thanks @juanpedromoreno, I'll do

File: build.sbt [hunk 1]
```diff
@@ -52,8 +58,8 @@ pgpSecretRing := file(s"$gpgFolder/secring.gpg")
 
 // General Settings
 lazy val commonSettings = Seq(
-  organization := "io.frees",
-  scalaVersion := "2.12.6",
+  organization := "higherkindness",
```
[the remark this answers is not in the document]
Comment (franciscodr) [reply build.sbt]: I have put this value based on the link that @juanpedromoreno posted above. Maybe he could confirm if the value is correct

[the remark this answers is not in the document]
Comment (juanpedromoreno) [reply build.sbt]: Good question, `mu` is set up as `organization := "higherkindness"`, and it has been successfully published. I'm guessing `higherkindness` would be ok

[the remark this answers is not in the document]
Comment (pepegar) [reply build.sbt]: Good then! However I think we should double-check how we want to publish all artifacts, if under io.higherkindness or higherkindness, but all shall have the same :). I had a conversation with @andyscott over slack about it, but don't recall the specifics.

[the remark this answers is not in the document]
Comment (juanpedromoreno) [reply build.sbt]: `mu` and `droste` are under `io.higherkindness`: https://repo1.maven.org/maven2/io/higherkindness/.

@pepegar is that what you mean?

[the remark this answers is not in the document]
Comment (pepegar) [reply build.sbt]: Ah cool, yeah, I referred to that! My brain is vacationing, sorry ;)

[the remark this answers is not in the document]
Comment (juanpedromoreno) [reply build.sbt]: No worries :)

**Pull request merged**